### ISSUE TITLE
how to get Last searches from the Klevu SDK

### ISSUE THREAD

**Issue by iamyashkhandelwal** [klevultd/frontend-sdk#146]
How can we get the last searches data from the Klevu SDK?
How will we get the data from this handler function and when this function will get updated ?
<img width="511" alt="image" src="https://user-images.githubusercontent.com/43111324/200182002-4db75844-4b80-40b6-a862-627999267933.png">

**Comment by rallu** [comment]
Yes. We seem to be missing that from documentation. We should add it. 

This is how you do it:
```
import { KlevuLastSearches } from "@klevu/core"
const theSearches = KlevuLastSearches.get()
```

This list gets updated when ever the analytical call for search is done.

**Comment by iamyashkhandelwal** [comment]
I tried this way as well @rallu . But I'm getting empty array till now

Output:
<img width="949" alt="image" src="https://user-images.githubusercontent.com/43111324/200250881-8688215b-157b-4b1a-a7ef-7bfd04f5d6a9.png">

Code: 
<img width="656" alt="image" src="https://user-images.githubusercontent.com/43111324/200250992-65306f7f-4168-471f-9ac1-7fe4a85f00f4.png">

**Comment by rallu** [comment]
Have you implemented click events from results? You need to search and click some products with everything properly implemented in order the last searches to populate.

**Comment by iamyashkhandelwal** [comment]
Okay! got it @rallu 
Thank you
